clients/drdisas/drdisas.cpp

```diff
@@ -159,6 +131,25 @@ main(int argc, const char *argv[])
     }
 #endif
 
+    // i#4021: arm not yet supported.
```

Comment:
Shown here.

Reply:
OK but IMHO it would best to also have one up in the option list.

Reply:
kk